docker/server/build.gradle

```diff
@@ -1,17 +1,25 @@
+import io.deephaven.tools.docker.Architecture
 import io.deephaven.tools.License
 
 plugins {
     id 'com.bmuschko.docker-remote-api'
 }
 
-def baseMap = [
-    'server-base': 'server',
-    'nltk-base': 'server-nltk',
-    'pytorch-base': 'server-pytorch',
-    'sklearn-base': 'server-sklearn',
-    'tensorflow-base': 'server-tensorflow',
+def hostArch = Architecture.fromHost()
+
+def baseMapAmd64 = [
+        'server-base': 'server',
+        'nltk-base': 'server-nltk',
+        'pytorch-base': 'server-pytorch',
+        'sklearn-base': 'server-sklearn',
+        'tensorflow-base': 'server-tensorflow',
 ]
 
+// Only the server image is supported on arm64
+def baseMapArm64 = baseMapAmd64.findAll { key, value -> value == 'server' }
```

Comment:
this is very unfortunate - do none of these libraries support arm yet?

Reply:
I haven't spent the time yet trying to matrix them out yet.

Reply:
TODO and issue for followup?